ui/src/app/views/account-creation/component.html

```diff
@@ -14,11 +14,27 @@ <h2>Create your account</h2>
       </div>
     </div>
     <div class="form-section">
-      <input type="text" class="short-input" id="username" name="username" [(ngModel)]="profile.username" (ngModelChange)="usernameChanged()" placeholder="Create Account">
-      <h5 style="display: inline-block;">@{{gsuiteDomain}}</h5>
+      <input type="text" class="short-input" id="username" name="username" [(ngModel)]="profile.username" (ngModelChange)="usernameChanged()" placeholder="New Username">
+      <h5 style="display: inline-block;" class="domain">@{{gsuiteDomain}}</h5>
+      <clr-tooltip>
+        <clr-icon clrTooltipTrigger shape="info-standard" class="is-solid info-icon"></clr-icon>
+        <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
+          <div>
+            Usernames can contain only letters (a-z), numbers (0-9), dashes (-), underscores (_),
+            apostrophies ('), and one period (.) in a row (maximum of 64 characters).
+          </div>
+          <br>
+          <div>
+            Usernames can begin or end with non-alphanumeric characters except (.).
```

Comment:
I guess to skip the middle-man: @lstrano-broad

I think this would be simplified as:
- Usernames can contain only letters (a-z), numbers (0-9), dashes (-), underscores (_), apostrophes ('), and periods (.) (maximum of 64 characters).
- Usernames cannot begin or end with a period (.) and may not contain more than one period (.) in a row.

Reply:
I approve of this new copy - thx